plat/allwinner/common/sunxi_bl31_setup.c

```diff
@@ -34,6 +35,17 @@ void bl31_early_platform_setup2(u_register_t arg0, u_register_t arg1,
 	console_16550_register(SUNXI_UART0_BASE, SUNXI_UART0_CLK_IN_HZ,
 			       SUNXI_UART0_BAUDRATE, &console);
 
+#ifdef BL32_BASE
+	/* Populate entry point information for BL32 */
+	SET_PARAM_HEAD(&bl32_image_ep_info, PARAM_EP, VERSION_1, 0);
+	SET_SECURITY_STATE(bl32_image_ep_info.h.attr, SECURE);
+	/*
+	 * Tell BL32 where the trusted software image
+	 * is located and the entry state information
+	 */
```

Comment:
This comment is a bit weird, I don't think we tell BL32 anything here, it's BL31 that reads this data. So can you please just remove it? The comment above should be sufficient.